pom.xml

```diff
@@ -276,6 +276,10 @@
             <id>vankka</id>
             <url>https://nexus.vankka.dev/repository/maven-public/</url>
         </repository>
+        <repository>
+            <id>dynmap</id>
+            <url>http://repo.mikeprimm.com/</url>
```

Comment:
Probably should be added to / proxied with nexus.scarsz.me @Scarsz (I'm not able to proxy repos/opinion)